gym/wrappers/rescale_action.py

```diff
@@ -0,0 +1,28 @@
+import numpy as np
+
+import gym
+from gym import spaces
+
+
+class RescaleAction(gym.ActionWrapper):
+    r"""Rescale the continuous action space of the environment from a range of [a, b]. """
+    def __init__(self, env, a, b):
+        assert isinstance(env.action_space, spaces.Box), 'expected Box action space.'
```

Comment:
It's generally nice to show the unexpected value for when the assert fails:
```suggestion
        assert isinstance(env.action_space, spaces.Box), (
            "expected Box action space, got {}".format(type(env.action_space)))
```

Reply:
Thanks a lot ! @hartikainen